src/pages/Profile.tsx

```diff
@@ -1,5 +1,219 @@
+    marginTop: 2,
+  },
+  followingcountbtn: {
+    border: "none",
+    fontSize: 15,
+    padding: 0,
+    marginLeft: 5,
+    color: "black",
+    backgroundColor: "white",
+    textTransform: "none",
+    marginTop: 2,
+  },
+  bioDisplay: {
+    marginBottom: 20,
+  },
+  parentBox: {
+    width: 650,
+  },
+  avatarBannerBox: {
+    height: 200,
+  },
+  profileInfoBox: {
+    paddingLeft: 15,
+    paddingTop: 20,
+  },
+  calendarBox: {
+    alignItems: "center",
+    display: "flex",
+  },
+};
+
 const Profile = () => {
-  return <h1 style={{ margin: "auto" }}>Profile</h1>;
+  const navigate = useNavigate();
+  const [value, setValue] = React.useState("one");
+  const [profileContents, setProfileContents] = React.useState({
+    tweetcount: "",
+    bio: "",
+    joindate: "",
+  });
+  const user = useAppSelector((state) => state.user);
+
+  useEffect(() => {
+    const fetchProfileContents = async () => {
+      const result = await axios.get(
+        "http://localhost:3001/api/profile/getProfileContents",
+        {
+          params: {
+            userId: user.userId,
+          },
+        }
+      );
+      setProfileContents(result.data);
+    };
+
+    fetchProfileContents();
+  }, [value, user]);
+  return (
+    <>
+      <Box style={styles.parentBox}>
+        <Box>
+          {/**Not sure about this */}
+          <Box style={styles.header}>
+            <IconButton style={styles.arrowBtn} onClick={() => navigate(-1)}>
+              <KeyboardBackspaceIcon color="secondary" />
+            </IconButton>
+            <Box>
+              <Typography sx={styles.arrowText}>{user.username}</Typography>
+              <Typography sx={styles.tweetCountStyle}>
+                {profileContents.tweetcount} Tweets
+              </Typography>
+            </Box>
+          </Box>
+          <Box sx={styles.avatarBannerBox}>
+            <Box
+              component="img"
+              sx={{ position: "relative", width: "650px", height: "200px" }}
+              src={process.env.PUBLIC_URL + "/blue.jpg"}
+              alt="temp"
+            />
+            <Avatar
+              alt="profile picture"
+              src={process.env.PUBLIC_URL + "/rock.jpg"}
+              sx={styles.avatar}
+            />
+          </Box>
+          <Button
+            variant="contained"
+            startIcon={<EditIcon />}
+            sx={styles.editBtn}
+          >
+            Edit Profile
+          </Button>
+
+          <Box style={styles.profileInfoBox}>
+            <Typography variant={"h2"} style={styles.displayName}>
+              {user.displayName}
+            </Typography>
+            <Typography variant={"h3"} style={styles.usernameDisplay}>
+              @{user.username}
+            </Typography>
+            <Typography style={styles.bioDisplay}>
+              {profileContents.bio}
+            </Typography>
+            <Box style={styles.calendarBox}>
+              <CalendarMonthIcon></CalendarMonthIcon>
```

Comment:
Can we please make components without children self closing?
```suggestion
              <CalendarMonthIcon/>
```

Reply:
agree